client/CBitmapHandler.cpp

```diff
@@ -18,17 +18,17 @@
 
 namespace BitmapHandler
 {
-	SDL_Surface * loadH3PCX(ui8 * data, size_t size);
+SDL_Surface * loadH3PCX(ui8 * data, size_t size);
 
-	SDL_Surface * loadBitmapFromDir(std::string path, std::string fname, bool setKey=true);
+SDL_Surface * loadBitmapFromDir(std::string path, std::string fname, bool setKey = true);
 }
 
-bool isPCX(const ui8 *header)//check whether file can be PCX according to header
+bool isPCX(const ui8 * header) //check whether file can be PCX according to header

```

Comment:
Spacebar before comment - interesting change, i think code looks bit more tidy with it.